packages/Header/Header.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div id="header" pt3 pb3 mb3>
+    <div v-if="!noPicture" class="container" flex row mobile-column-reverse align-center space-between wrap>
+      <div half>
+        <h1 mb1>{{title}}</h1>
+        <p mb2>{{description}}</p>
+        <slot v-if="$slots.default" />
+      </div>
+      <div half>
+        <img v-if="imgUrl" :src="imgUrl" :alt="title" />
+        <slot name="items"></slot>
+      </div>
+    </div>
+    <div v-else class="container">
+      <div>
+        <h1 mb1>{{title}}</h1>
+        <p mb2>{{description}}</p>
+      </div>
+      <slot v-if="$slots.default" />
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Header',
+  props: {
+    title: { type: String },
+    description: { type: String },
+    noPicture: { type: Boolean, default: false },
```

Comment:
`noPicture` could be computed based on `imgUrl` presence.
Also, one is img, the other one picture. We need consistency either both img or both picture but let's not mix